resources/public/api/conf/2.0/application.raml

```diff
@@ -372,4 +372,40 @@ traits:
                       notFound:
                         description: The requested message does not exist, has been archived, or is not available to the EORI number
                         value:
-                          code: NOT_FOUND
\ No newline at end of file
+                          code: NOT_FOUND
+  /arrivals:
+    post:
+      displayName: Send an arrival notification message
+      description: |
+        Notify the Customs Office of Destination that a transit movement has arrived at its final destination.
+
+        This request is also called an ‘Arrival Notification’ E_ARR_NOT (IE007) message.
+
+        A successful response will contain a URI and an arrival ID for the notification, which you can use to retrieve the message status.
+
+        It will also contain an embedded request ID, box ID and box name, which you can use to match this movement to any messages received from the Push Pull Notifications API.
```

Comment:
No it doesn't, not this time -- traders can (and will) get this themselves. Remove this line.

Reply:
removed